citesphere-importer/src/main/java/edu/asu/diging/citesphere/importer/core/zotero/template/impl/JsonGenerationService.java

```diff
@@ -41,8 +42,11 @@ public ObjectNode generateJson(JsonNode template, BibEntry entry) {
         ItemJsonGenerator generator = generators.get(entry.getArticleType());
         if (generator != null) {
             return generator.generate(template, entry);
+        } else if (entry instanceof Publication){
+            generator = generators.get("CrossRef");
+            return generator.generate(template, entry);
```

Comment:
what is this code for?

Reply:
Removed this block, as crossref generator is removed.